build-tools/feeds/README.md

```diff
@@ -0,0 +1,10 @@
+The files in this folder are lists of packages that should be published to the four npm feeds that we publish to. They
```

Comment:
I think these data should be per "release group".  it shouldn't be under `build-tools`

Reply:
agreed, but I can't do that until we release build-tools and client consumes it. I'll just remove these changes for now and add them after build-tools is upgraded.